src/main/java/ua/tc/marketplace/model/entity/Article.java

```diff
@@ -0,0 +1,76 @@
+package ua.tc.marketplace.model.entity;
+
+import java.util.ArrayList;
+import lombok.*;
+import jakarta.persistence.*;
+import java.time.LocalDateTime;
+import java.util.List;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import ua.tc.marketplace.model.enums.ArticleStatus;
+
+@Entity
+@Table(name = "article")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Article {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Column(nullable = false)
+  private String title;
+
+  @Column(nullable = false, columnDefinition = "TEXT")
+  private String content;
+
+  @ManyToOne
+  @JoinColumn(name = "author_id", nullable = false)
+  private User author;
+
+  @CreationTimestamp
+  @Column(updatable = false)
+  private LocalDateTime createdAt;
+
+  @UpdateTimestamp
+  private LocalDateTime updatedAt;
+
+  @Builder.Default
+  @ManyToMany
```

Comment:
Shouldn't it be OneToMany? One article and many photos. Or you suppose that authors can use photos from other articles?